### DISCUSSION TITLE
Add summernote editor

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6508] [Q&A]
I am trying to use summernote editor but it is not working fine. How to use the editor in statamic please ?

**Comment by jasonvarga** [comment]
https://statamic.dev/extending/fieldtypes
https://summernote.org/getting-started/

**Reply by HazemMohamed252** [reply]
It asks me to include bootstrap css and jquery. Please how to appeand it as when i am trying to append the page style be changed

**Reply by jasonvarga** [reply]
jquery is already in the control panel. 
You can add css as explained here https://statamic.dev/extending/control-panel#adding-css-and-js-assets

Please try to use the documentation.

**Reply by HazemMohamed252** [reply]
Does bootstrap js and css exists on statmic or not ?

**Reply by jasonvarga** [reply]
No

**Reply by HazemMohamed252** [reply]
Not working as explained. I want it for the cp not the front

**Reply by jasonvarga** [reply]
You can't just say "it's not working". You'll need to provide more details. What exactly isn't working? Are there error messages, etc.
